Doc/whatsnew/3.14.rst

```diff
@@ -1437,6 +1437,11 @@ Build changes
   with :c:expr:`Py_NO_LINK_LIB`. (Contributed by Jean-Christophe
   Fillion-Robin in :gh:`82909`.)
 
+* building with clang-cl on Windows now supports PGO (profile guided
+  optimization), uses ``-flto=thin`` and can be configured to use
```

Comment:
```suggestion
  optimization), and can be configured to use
```

Is using `-flto=thin` important to call out? It shouldn't change the ABI of the build output at all, and it's not configurable.